<launch>

  <arg name="contexts_json" />

  <node pkg="riberry_startup" name="keyword_extraction" type="keyword_extraction.py">
    <rosparam subst_value="true">
      contexts_json: $(arg contexts_json)
    </rosparam>
  </node>

  <node pkg="riberry_startup" name="keyword_to_action" type="keyword_to_action.py" />

</launch>
